portfolio/application_default_credentials.json

```diff
@@ -0,0 +1,7 @@
+{
+  "client_id": "764086051850-6qr4p6gpi6hn506pt8ejuq83di341hur.apps.googleusercontent.com",
```

Comment:
There are people that scrape github for api keys.  Is this key only usable on your particular project?